### ISSUE TITLE
Question about the method predict_one()

### ISSUE THREAD

**Issue by beinathan** [online-ml/river#908]
I used a MultiOutputMLP to train the model and got a metric of 0.012. But when I re-invoked the training data to make predictions about the trained model, I found that the y_pred was very different from y. The specific data is shown below.

-         x={'u': 0.0, 'v': 0.0, 'r': 0.0, 'tau_u': 20.0, 'tau_v': -20.0, 'tau_r': 2.32}
-         y={'out_0': 0.0, 'out_1': 0.0, 'out_2': 0.0}
-         y_pred={'out_0': 11.218229370225686, 'out_1': 7.723067205676228, 'out_2': 0.0}

 I don't feel that this matches the precision that I get from training. This is my question. Thanks a lot.

**Comment by MaxHalford** [comment]
Not sure what's happening! Could you share some code so we can reproduce?

**Comment by beinathan** [comment]
[code.zip](https://github.com/online-ml/river/files/8440020/code.zip)
Thanks for your reply, here is my code. You could run main.py directly, where the variable out_new is the predicted value, and  the variable out_real is the real value.
I tried to use model in the streaming data  directly rather than do pre-training with collected datas. This avoids the above problem, but when the system is stable, the predicted value is always 0, so may this be the correct result? Thanks a lot!

**Comment by beinathan** [comment]
Hello again, when I tried to change to a single output task, I found that the prediction value obtained by using the method predict_one() is still 0. I'd like to ask how to get the correct prediction value using by the method predict_one() after building the model. I guess my use is incorrect, please can you point out my problem?

**Comment by MaxHalford** [comment]
Is this still an issue @beinathan? I haven't looked at it yet.

**Comment by beinathan** [comment]
I'm sorry, I've solved the problem. Thanks for your patience.
